api/v1alpha1/lvmcluster_types.go

```diff
@@ -45,6 +59,14 @@ type DeviceClass struct {
 	// +kubebuilder:validation:Pattern="^[a-z0-9]([-a-z0-9]*[a-z0-9])?$"
 	Name string `json:"name,omitempty"`
 
+	// VolumeGroupName is the name of the volume group to be created.
+	// Validations to confirm that this field can be used as metadata.name field in storageclass
+	// ref: https://github.com/kubernetes/apimachinery/blob/de7147/pkg/util/validation/validation.go#L209
+	// +kubebuilder:validation:MaxLength=245
+	// +kubebuilder:validation:MinLength=1
+	// +kubebuilder:validation:Pattern="^[a-z0-9]([-a-z0-9]*[a-z0-9])?$"
+	VolumeGroupName string `json:"volumeGroupName,omitempty"`
```

Comment:
I'm not convinced that this field is required just yet.

Reply:
removed. Only supporting one pool per device class/ Volume group now.